lib/shortcodes/faq/faq.php

```diff
@@ -1,6 +1,6 @@
 <?php namespace CAHNRSWP\Plugin\Pagebuilder;
 
-?><dl class="cpb-faq">
+?><dl class="cpb-faq <?php echo esc_html( $classes ); ?>">
```

Comment:
`esc_html()` should work here, but `esc_attr()` is more dependable when outputting HTML attributes.